### PR TITLE
complete app

### PR DESCRIPTION
**Pull request by phgupta**
Completed energy baselines app.

### CODE REVIEW

File: energy_baselines/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+googleapis-common-protos==1.5.8
```
Comment (gtfierro) [inline energy_baselines/requirements.txt]: I believe quite a few of these are just `pymortar`'s requirements, and aren't generally used. Can you remove the `pymortar` requirements to that those are more flexible to change? I think you can just have scipy, sklearn, scikit-learn, pymortar and pandas.

**Review by gtfierro — CHANGES_REQUESTED** [review]

**Comment by phgupta** [comment]
Removed the pymortar's requirements.

**Comment by gtfierro** [comment]
Great! 

I was thinking given how many energy baseline apps we are going to have, do you mind moving your files into an application directory within a more general "energy_baselines" directory? I think this might help with organization, but let me know what you think

**Comment by phgupta** [comment]
Yes, you're right. I named the application directory "baselines" in the event that we add other types of baselines like power/occupancy baselines.

**Comment by gtfierro** [comment]
Great, thanks!

**Review by gtfierro — APPROVED** [review]

**Pull request merged**